app/models/setting.rb

```diff
@@ -259,6 +259,12 @@ def self.bypass_readonly(s, &block)
     s.readonly! if old_readonly
   end
 
+  def self.replace_keywords(keyword)
+    if keyword.respond_to?(:gsub)
+      keyword.gsub '$VERSION', SETTINGS[:version].version
+    end
+  end
```

Comment:
I find this method a bit unsafe, when it's called with something else than string, then it returns `nil`. Can you change it to return actually the unchanged argument if it does not respond to `gsub`?